src/WebJobs.Script.WebHost/Controllers/ExtensionBundleController.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Script.BindingExtensions;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Azure.WebJobs.Script.ExtensionBundle;
+using Microsoft.Azure.WebJobs.Script.Models;
+using Microsoft.Azure.WebJobs.Script.Properties;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization.Policies;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Controllers
+{
+    [Authorize(Policy = PolicyNames.AdminAuthLevel)]
+    public class ExtensionBundleController : Controller
+    {
+        private readonly IExtensionBundleContentProvider _extensionBundleContent;
+
+        public ExtensionBundleController(IExtensionBundleContentProvider extensionBundleContent)
+        {
+            _extensionBundleContent = extensionBundleContent ?? throw new ArgumentNullException(nameof(extensionBundleContent));
+        }
+
+        [HttpGet]
+        [Route("admin/host/extensionBundle/staticcontent/v1/templates")]
```

Comment:
The `staticcontent` feels out of place here. Even if the file path has this in it behind-the-scenes, can we remove it for simplicity?